CMakeLists.txt

```
@@ -53,6 +53,7 @@ if(FIND_PM_DEPS)
   find_package(ArgumentParser CONFIG REQUIRED)
   find_package(SwiftCrypto CONFIG REQUIRED)
   find_package(SwiftDriver CONFIG REQUIRED)
+  find_package(SwiftCollections CONFIG REQUIRED)
```

```suggestion
  find_package(swift-collections CONFIG REQUIRED)
```

The naming here is due to the file name for the generated files